CHANGES.md

```diff
@@ -24,6 +24,15 @@ OpenSSL 3.0
 
 ### Changes between 1.1.1 and 3.0 [xx XXX xxxx] ###
 
+ *) Add CAdES-BES signature verification support, mostly derived
```

Comment:
`*)` -> `*`  (see existing entries below)

Reply:
When removing the `)`, please note also that the entire paragraph needs to be unindented by one space.

Reply:
Fixed